hugo/content/documentation/user/mumble-url/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Mumble URL"
+---
+Mumble defines it's own `mumble://` URL scheme to share links to servers and channels and to be able to connect to them directly.
+
+On Windows the installer will register `mumble://` links to be opened with Mumble.
+
+Mumble links can be opened on the command line by passing them to mumble.exe too:
+
+```
+mumble.exe mumble://example.com/?version=1.2.0
+```
+
+or with a channel path:
+
+```
+mumble.exe mumble://example.com/ChanA/ChanB?version=1.2.0
+```
+
+You can get get a URL in the Channel context menu (right click):
+
+![Screenshot of Mumble channel context menu Copy URL item](screenshot-chan-copy-url.jpg)
+
+## Connect Behavior
+
+When opening a Mumble link the Mumble client will connect to the server.
+
+* If you are already registered you will connect with your account
+* If the URL contains a username that username will be used
+* If no username is determined, the client will ask with what name you want to connect with
+
+## URL Format
+
+```
+mumble://[username[:password]@]<address>[:port]/[channelpath][?version=<serverversion>][&title=<servername>][&url=<serverurl>]
+```
+
+* optional `username` and `password`
+* mandatory `address` is mandatory; a hostname, IPv4, or IPv6 address
+* optional `port`; defaults to 64738
+* optional `channelpath`; specifies a channel subtree to join; if the Mumble client can join the server but not the channel it will stay in the default channel
+* optional `serverversion` - protocol version of the server; Should no longer be necessary; Relevant for backwards compatibility to pre-1.2.0 versions

```

Comment:
Is that really the server's version? I was thinking this was the scheme's version :eyes:

Reply:
I think the URL scheme changed with Mumble version 1.2 hence the 1.2.0 specifier. But has been like this since.

I actually think we can and should drop all mentions of the version specifier in the main/generic documentation and only add a deprecated / old section mentioning it. 1.2 is so very old.

Reply:
I guess it makes sense here in the main format desc - but we could drop it from the examples etc

Reply:
Or was it a mumble protocol incompatibility that made this necessary? I don't remember

Reply:
https://www.mumble.info/blog/version-numbering-change/

Reply:
From the wiki page

> Note that also beginning with 1.2.4, "?version" is not required, but it's still recommended that you include it as anyone who has 1.2.3 or earlier will launch the 1.1.x compatibility client when clicking links without "?version=1.2.0". 

1.2.4 was released in 2014 https://www.mumble.info/blog/mumble-1.2.4-released/

so I'd consider the version param non-essential to describe and only mention it secondarily

Reply:
Yeah, I agree that we can probably drop the version from pretty much anything :+1: